src/main/scala/com/fulcrumgenomics/bam/ClipOverlappingReads.scala

```diff
@@ -59,21 +59,25 @@ class ClipOverlappingReads
   @arg(flag="o", doc="Output SAM or BAM file.") val output: PathToBam,
   @arg(flag="s", doc="Soft clip reads instead of hard clipping.") val softClip: Boolean = false,
   @arg(flag="r", doc="Reference sequence fasta file.") val ref: PathToFasta,
-  @arg(flag="a", doc="Automatically clip extended attributes that are the same length as bases.") val autoClipAttributes: Boolean = false
+  @arg(flag="a", doc="Automatically clip extended attributes that are the same length as bases.") val autoClipAttributes: Boolean = false,
+  @arg(flag="m", doc="The maximum # of reads in memory for any internal sorter") val maxReadsInMemory: Option[Int] = None,
+  @arg(flag="t", doc="SAM tags to remove from the input") val tagsToRemove: Seq[String] = Seq.empty
```

Comment:
I really don't like having this here.  I feel like it has nothing to do with clipping overlapping reads, and is a first step down a picard-like path of adding odd-ball options to tools to workaround other problem (e.g. memory usage).  If the tags are really a substantial part of the memory, why not just pipe the data through a tool to strip out tags first, instead of building that in here (and potentially elsewhere)?